### PR TITLE
Get all cross-sell data from PCMS

### PR DESCRIPTION
**Pull request by cpiehl1**
# Jira Issue: []

## What?
- Get all cross-sell data from PCMS

## Why?
- This simplifies the required giraffe logic a lot

## Optional checklist
- [x] Unit tests written
- [x] Tested locally
- [ ] Codescouted

### CODE REVIEW

**Review by hugokallstrom — APPROVED** [review]
Soooo good!!

File: src/resolvers/uncategorised/crossSell.ts [hunk 1]
```diff
@@ -1,249 +1,69 @@
...
+      return {
+        type: crossSellData.productType as CrossSellType,
+        contractType: typeOfContract as TypeOfContract,
+        title: crossSellData.notificationTitle,
+        description: crossSellData.notificationDescription,
+        callToAction: crossSellData.notificationCta,
+        imageUrl: coverPhoto.coverPhotoUrl,
+        blurHash: coverPhoto.coverPhotoBlurHash,
+        info: {
+          displayName: strings(`CONTRACT_DISPLAY_NAME_${typeOfContract}`),
```
Comment (hugokallstrom) [inline src/resolvers/uncategorised/crossSell.ts:43]: We should be able to remove this. @sampettersson are you using this field on iOS?

Comment (sampettersson) [reply src/resolvers/uncategorised/crossSell.ts:43]: We seem to use this field on iOS actually, should we not?

Comment (hugokallstrom) [reply src/resolvers/uncategorised/crossSell.ts:43]: We are using title, should be the same as contract name. Specified in PCMS :)

Comment (hugokallstrom) [reply src/resolvers/uncategorised/crossSell.ts:43]: Can we deprecate `displayName` Carl?

Comment (cpiehl1) [reply src/resolvers/uncategorised/crossSell.ts:43]: Sure!

**Pull request merged**